app/controllers/prison/visits_controller.rb

```diff
@@ -9,18 +9,26 @@ def process_visit
     @visit = load_visit.decorate
   end
 
+  # rubocop:disable Metrics/MethodLength
   def update
     booking_response = booking_responder.respond!
+
     if booking_response.success?
-      flash[:notice] = t('process_thank_you', scope: %i[prison flash])
+      flash[:notice] = t('process_thank_you', scope: [:prison, :flash])
```

Comment:
@StupidCodeFactory here it has also changed from `%i` to explicit array. I'm pretty sure is rubocop